src/gen_rpc_app.erl

```diff
@@ -35,7 +35,8 @@ stop(_State) ->
 %%% ===================================================
 -spec start() -> 'ok' | {'error', term()}.
 start() ->
-    application:start(?APP).
+    application:ensure_all_started(?APP),
+    start(normal, []).
```

Comment:
I was not thinking straight. 
The error was in run_target.sh use of  'erl -s' to bootstrap. Erl -s defaults the funciton as start, so it is calling gen_rpc_app:start but the what required is applcation:start and bootstrapping of other deps.